diff -up x265_2.9/source/CMakeLists.txt.0000~ x265_2.9/source/CMakeLists.txt
--- x265_2.9/source/CMakeLists.txt.0000~	2019-01-08 16:55:04.199575988 +0100
+++ x265_2.9/source/CMakeLists.txt	2019-01-08 16:56:35.348462975 +0100
@@ -55,6 +55,25 @@ if("${SYSPROC}" STREQUAL "" OR X86MATCH
         add_definitions(-DX86_64=1)
         message(STATUS "Detected x86_64 target processor")
     endif()
+elseif(${SYSPROC} MATCHES "armv5.*")
+    message(STATUS "Detected ARMV5 system processor")
+    set(ARMV5 1)
+    add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=0 -DHAVE_NEON=0)
+elseif(${SYSPROC} STREQUAL "armv6l")
+    message(STATUS "Detected ARMV6 system processor")
+    set(ARM 1)
+    set(ARMV6 1)
+    add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=1 -DHAVE_NEON=0)
+elseif(${SYSPROC} STREQUAL "armv7l")
+    message(STATUS "Detected ARMV7 system processor")
+    set(ARM 1)
+    set(ARMV7 1)
+    add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=1 -DHAVE_NEON=1)
+elseif(${SYSPROC} STREQUAL "aarch64")
+    message(STATUS "Detected AArch64 system processor")
+    set(ARM 1)
+    set(ARMV7 1)
+    add_definitions(-DX265_ARCH_ARM=1 -DHAVE_ARMV6=1 -DHAVE_NEON=1)
 elseif(POWERMATCH GREATER "-1")
     message(STATUS "Detected POWER target processor")
     set(POWER 1)
diff -up x265_2.9/source/common/cpu.cpp.0000~ x265_2.9/source/common/cpu.cpp
--- x265_2.9/source/common/cpu.cpp.0000~	2018-10-05 14:14:40.000000000 +0200
+++ x265_2.9/source/common/cpu.cpp	2019-01-08 16:55:04.199575988 +0100
@@ -37,7 +37,7 @@
 #include <machine/cpu.h>
 #endif
 
-#if X265_ARCH_ARM && !defined(HAVE_NEON)
+#if X265_ARCH_ARM && (!defined(HAVE_NEON) || HAVE_NEON==0)
 #include <signal.h>
 #include <setjmp.h>
 static sigjmp_buf jmpbuf;
