--- x265-10376/source/test/CMakeLists.txt.0002~	2015-05-06 18:03:56.946061580 +0200
+++ x265-10376/source/test/CMakeLists.txt	2015-05-06 18:04:34.344331005 +0200
@@ -22,7 +22,7 @@ add_executable(TestBench ${YASM_SRC}
     mbdstharness.cpp mbdstharness.h
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     set_target_properties(TestBench PROPERTIES LINK_FLAGS ${LINKER_OPTIONS})
 endif()
