--- x265-11300/source/test/CMakeLists.txt.0002~	2016-02-12 13:49:49.322716930 +0100
+++ x265-11300/source/test/CMakeLists.txt	2016-02-12 13:50:15.621707778 +0100
@@ -44,7 +44,7 @@ add_executable(TestBench ${YASM_SRC}
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
 
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     if(EXTRA_LIB)
         list(APPEND LINKER_OPTIONS "-L..")
