--- x265-10987/source/test/CMakeLists.txt.0002~	2015-09-12 15:52:19.637102753 +0200
+++ x265-10987/source/test/CMakeLists.txt	2015-09-12 15:52:44.197169192 +0200
@@ -23,7 +23,7 @@ add_executable(TestBench ${YASM_SRC}
     mbdstharness.cpp mbdstharness.h
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     if(EXTRA_LIB)
         list(APPEND LINKER_OPTIONS "-L..")
